README.md

```diff
@@ -1,18 +1,78 @@
-# Compiling Protocol Buffers
+<div align="center">
```

Comment:
Fancy 😏

Reply:
Guess where I got it from: https://github.com/wgslr/encrypt-to-me-rage/blob/master/README.md

Reply:
Hahaha. And I didn't even write that:P